build.sbt

```diff
@@ -1,14 +1,14 @@
 name := "akka-xlsx"
 
-lazy val AlpakkaXmlVersion = "3.0.3"
-lazy val AkkaTestkitVersion = "2.6.16"
-lazy val ScalatestVersion = "3.2.3"
-lazy val ScalatestPlusVersion = "3.2.3.0"
-lazy val ScalacheckVersion = "1.15.2"
+lazy val PekkoXmlVersion = "1.0.2"
+lazy val PekkoStreamVersion = "1.0.3" //fixes statefulMapConcat issue from pekko-stream 1.0.2
```

Comment:
It shows why we needed to override a transitive dependency. If there was no bug in 1.0.2, there would be no version needed at all - it would be pulled by pekko-xml.

Reply:
ok, makes sense. It should be clearly stated that it overrides a transitive dependency then